nx/lib/nx/shared.ex

```diff
@@ -361,7 +361,7 @@ defmodule Nx.Shared do
         {"reverse square root", quote(do: Complex.divide(1, Complex.sqrt(var!(x)))),
          "$$rsqrt(z) = \\frac{1}{\\sqrt{z}}$$"},
       cbrt:
-        {"cube root", quote(do: Complex.power(var!(x), 1 / 3)), "$$cbrt(z) = z^{\\frac{1}{3}}$$"},
+        {"cube root", quote(do: Complex.cbrt(var!(x))), "$$cbrt(z) = z^{\\frac{1}{3}}$$"},
```

Comment:
There are some fixes that need to be released on Complex for this PR to work on CI. Currently they are only present locally for me.